src/modals/SettingsModal.module.scss

```diff
@@ -0,0 +1,9 @@
+:local {
+  .container {
+    padding: 0.7143rem;
```

Comment:
Why the different rounding on line 3 and 7?

Reply:
🤷‍♂️